src/components/learner-credit-management/BudgetDetailCatalogTabContents.jsx

```diff
@@ -48,12 +43,20 @@ const BudgetDetailCatalogTabContents = () => {
             indexName={configuration.ALGOLIA.INDEX_NAME}
             searchClient={searchClient}
           >
-            <CatalogSearch catalogUuid={subsidyAccessPolicy?.catalogUuid} />
+            <CatalogSearch enterpriseSlug={enterpriseSlug} />
           </InstantSearch>
         </SearchData>
       </Col>
     </Row>
   );
 };
 
-export default BudgetDetailCatalogTabContents;
+BudgetDetailCatalogTabContents.propTypes = {
+  enterpriseSlug: PropTypes.string.isRequired,
+};
+
+const mapStateToProps = state => ({
+  enterpriseSlug: state.portalConfiguration.enterpriseSlug,
+});
+
+export default connect(mapStateToProps)(BudgetDetailCatalogTabContents);
```

Comment:
Would recommend connecting the `CatalogSearch` component (or whatever child component actually _uses_ the `enterpriseSlug` prop) to the Redux store with `mapStateToProps` as opposed to connecting its parent and passing `enterpriseSlug` via props to its children.

This suggested change would avoid any prop drilling of `enterpriseSlug` between components.

Reply:
Thanks for the suggestion -- this is implemented in updated pr